src/modules-load/modules-load.c

```diff
@@ -81,10 +81,10 @@ static int apply_file(struct kmod_ctx *ctx, const char *path, bool ignore_enoent
                 char *l;
                 int k;
 
-                r = read_line(f, LONG_LINE_MAX, &line);
-                if (r < 0)
+                k = read_line(f, LONG_LINE_MAX, &line);
+                if (k < 0)
                         return log_error_errno(errno, "Failed to read file '%s': %m", path);
```

Comment:
oh, i fucked this one, this should have errno → k now...

Reply:
Done.